components/resources/jobExplorer/jobList/JobListComponent.tsx

```diff
@@ -10,14 +16,12 @@ const JobGroup = ({ subheading, jobs }) => (
         <div
           key={job.title}
           className={`flex flex-col items-center rounded-lg p-6 shadow-md ${
-            (job.title.includes("Game Developer") ||
-              job.title.includes("Cyber Security Analyst") ||
+            ( job.title.includes("Cyber Security Analyst") ||
```

Comment:
you shouldn't have to list out all the jobs here. it should just be able to reference the disabledJobs array

Reply:
This code here was not written by me, but I can reference the disabled Jobs array so I don't repeat myself.

Reply:
Code now references disabled jobs array. If job.title is included in disabledJobs, then the link is grayed out. Otherwise the active links are shaded orange.